src/stories/molecules/inputFields/Autocomplete.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { FormControlProps, Form } from 'react-bootstrap';
+import featherIcons from "epfl-elements/dist/icons/feather-sprite.svg";
+import './autocomplete.css';
+
+type Item = {
+  label: string;
+  value: string;
+};
+
+interface AutocompleteProps {
+  suggestions: Item[];
+  selected?: Item[];
+  multiple?: boolean;
+  itemValue?: Item;
+  placeholder?: string;
+  isReadonly?: boolean;
+  onChange?: (selectedItems: Item[]) => void;
+}
+
+/**
+ * An autocomplete list.
+ * If single selection: the selected element is shown directly in the input field. The "itemValue" could be the
+ * initialization of this field.
+ * For multiple selections: selected elements are shown above the input list and it is possible to delete them.
+ * The "selected" prop contains all selected items.
+ */
+export const Autocomplete: React.FC<AutocompleteProps & FormControlProps> = ({
+   suggestions,
+   selected = [],
+   multiple = false,
+   itemValue,
+   placeholder = '',
+   isReadonly = false,
+  onChange
+  }: AutocompleteProps & FormControlProps) => {
+  const [filteredSuggestions, setFilteredSuggestions] = useState<Item[]>([]);
+  const [inputValue, setInputValue] = useState(itemValue ? itemValue.label : '');
+  const [selectedSuggestions, setSelectedSuggestions] = useState<Item[]>(selected ? selected : []);
```

Comment:
Likewise, I don't think you need a ternary operator here as the props initializer already provides a default value.